### ISSUE TITLE
How to update all columns on conflict

### ISSUE THREAD

**Issue by go-jet** [go-jet/jet#238]
### Discussed in https://github.com/go-jet/jet/discussions/237

<div type='discussions-op-text'>

<sup>Originally posted by **rsmidt** May 17, 2023</sup>
I'm trying to implement a basic upsert where I want to update all fields as soon as I hit a constraint, e.g.

```go
Users.
	INSERT(Users.AllColumns).
	MODEL(user).
	ON_CONFLICT(Users.ID).
	DO_UPDATE(
		SET(Users.AllColumns.SET(Users.EXCLUDED.AllColumns)),
	)
```

Unfortunately, this doesn't compile. Am I doing something wrong here? Is there maybe a better way to achieve the upsert?</div>

**Comment by DevX86** [comment]
@go-jet Bump :) Can you take a look at this? Having a `MODEL()` for updating all columns besides the primary key would be super useful. Or something similar to the implementation above.

**Comment by dragondgold** [comment]
Agreed, it would be super useful, right now we have to manually create all the `SET` declarations

**Comment by go-jet** [comment]
Support added in `Release v2.13.0`.
